rico.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: rico
+title: Rico
+order: '16'
+headerText:
+permalink: /rico/
+header:
+  image: rico
+  text: Rico by Karakun
+nav:
+  bottom: false
+---
+
+Implement applications while leveraging state-of-the art technologies, patterns and practices.
+
+## Overview
+
+Rico is an application framework that provides several functionalities and APIs to create enterprise applications. 
+This includes features like tracing or monitoring that are critical in developing applications, escpecially in modern cloud based and distributed application landscapes. 
+Thus, Rico provides generic APIs with specific implementation for the mostly used frameworks and toolkits in modern application development.
+
+Rico originates from the _Canoo Dolphin Platform_ which appears to be discontinued. Rico is an (originally) API compatible fork the we continue to develop and improve.
+
+### Rationale
+
+[//]: # ( #### Motivation for Rico )
+
+The motivation for us to create Rico is to avoid code for standard use-cases to be written over-and-over again. Thus, we created it as an extension to commonly used frameworks like Spring and Jakarta EE (Java EE) that is easily adoptable.
+
+The outcome is a framework that is build on top of well-known frameworks combined with commonly used components and prepared to be used with a very short warmup. 
+Rico helps you to build your applications in a short time letting you focus on the business requirements as it has standard solutions for common technical use cases like _Data Access_, _Logging_, _Monitoring_ and _Security_ *already build in* and well prepared to get *integrated in your environment*.
+
+To achieve this, Rico provides clean APIs to seperate complexity coming from the set of technologies required to build your solution from your actual code required to provide your business solution.
+
+Also, Rico provides a remoting layer that helps you to seperate UI framework specific code so that is a lot easier to switch UI implementations. As we see a very rapidly changing environment in UI technologies, this speration helps to protect your investment put into the solution build with Rico as it will be less dependent on a concrete UI technology.
```

Comment:
```suggestion
Also, Rico provides a remoting layer that helps you to seperate UI framework specific code so that is a lot easier to switch UI implementations or choose different technologies for every target platform. As we see a very rapidly changing environment in UI technologies, this speration helps to protect your investment put into the solution build with Rico as it will be less dependent on a concrete UI technology.
```